src/Rules/Misc/Helpers.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Axe.Windows.Core.Types;
+using System;
+using UIAutomationClient;
+
+namespace Axe.Windows.Rules.Misc
+{
+    static class Helpers
+    {
+        private static readonly Lazy<IUIAutomationElement> _desktop = new Lazy<IUIAutomationElement>(GetDesktopElement);
+        public static IUIAutomationElement Desktop => _desktop.Value;
+
+        private static IUIAutomationElement GetDesktopElement()
```

Comment:
This seems odd to me. Rules (so far) is not a data collector--it operates purely on provided data. Now it's reaching out to the system to collect its own data.